test/spec/NodeConnection-test.js

```diff
@@ -192,6 +192,42 @@ define(function (require, exports, module) {
             });
         });
         
+        it("should receive progress events from asynchronous commands", function () {
```

Comment:
@zaggino I ran the Live preview test for `Node connection` and the test is timing out at `should receive progress events from asynchronous commands` [win 8.1]. Any idea why?